week1/task_学生成绩管理系统0.1.cpp

```diff
@@ -0,0 +1,62 @@
+#include <stdio.h> 
+int Not (float score[],int n);

```

Comment:
```suggestion
int get_failed_student_count(float score[], int n);
```

1. Always use meaningful and descriptive names to improve code readability
2. In C language, I suppose we should NOT capitalize the first letter of function name
3. There two(or three or four) widely used naming style, one is camel-case style and the other is underline style. Let's use the underline style to keep align with [Linux kernel code](https://github.com/torvalds/linux).